pom.xml

```diff
@@ -136,7 +136,7 @@
     <dependency>
       <groupId>io.cdap.plugin</groupId>
       <artifactId>hydrator-common</artifactId>
-      <version>${hydrator.version}</version>
+      <version>${cdap.plugin.version}</version>
```

Comment:
Why is this change needed? In general, we use `hydrator.version` to refer to the version of the parent hydrator plugin artifact